### DISCUSSION TITLE
Order by deeply nested relation M:M:1:M

### DISCUSSION THREAD

**Discussion by gjlstop** [prisma/prisma#13630] [Q&A]
I'm trying to do an order by deeply nested relation Many -> Many -> 1 -> Many with the following schema:

```
model Collection {
  collectionId String  @id @default(uuid()) @db.Uuid
  cards        Cards[]
  ...
}

model Card {
  cardId         String          @id @default(uuid()) @db.Uuid
  collectionId   String?         @db.Uuid
  collection     Collection?     @relation(fields: [collectionId], references: [collectionId])
  vendorCardInfo VendorCardInfo?
  ...
}

model VendorCardInfo {
  cardId    String            @unique @db.Uuid
  card      Card              @relation(fields: [cardId], references: [cardId])
  cardOrder VendorCardOrder[]
  ...
}

model VendorCardOrder {
  orderId     String         @id @default(uuid()) @db.Uuid
  cardId      String         @db.Uuid
  cardInfo    VendorCardInfo @relation(fields: [cardId], references: [cardId])
  price       Decimal        @db.Decimal(78, 0)
  ...
}
```

and I'm trying to do this within the following code (but based on https://github.com/prisma/prisma/issues/5008#issuecomment-772483990  it may not be supported):
```
  prisma.collection.findMany({
    where: {
       ...
    },
    orderBy: {
      cards: {
        vendorCardInfo: {
          cardOrder: {
            price: "asc",
          }
        }
      },
    },
  });
```
Basically, I'd like to sort the collection (which has many cards which has one info record which has many order records) query results ordered by the price of an order.
```

**Comment by austincrim** [comment]
Hey @gjlstop 👋,

You are correct, I don't think this use case is supported. You would need to manually sort the results in your app code or use [`$queryRaw`](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#queryraw).


Feel free to open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) describing your use case.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
